presto-hive/src/main/java/com/facebook/presto/hive/parquet/predicate/ParquetPredicateUtils.java

```diff
@@ -162,17 +159,12 @@ private static DictionaryPage readDictionaryPage(byte[] data, ParquetCodecFactor
                 return null;
```

Comment:
will be safer to make the return type `Optional<ParquetDictionaryPage>`  instead of returning `null`.